packages/shared/src/porter.ts

```diff
@@ -6,20 +6,23 @@ import {
   RetrievalKit,
   TreasureMap,
 } from '@nucypher/nucypher-core';
-import axios, { AxiosResponse } from 'axios';
+import axios, {
+  AxiosRequestConfig,
+  AxiosResponse,
+  HttpStatusCode,
+} from 'axios';
 import qs from 'qs';
 
 import { Base64EncodedBytes, ChecksumAddress, HexEncodedBytes } from './types';
 import { fromBase64, fromHexString, toBase64, toHexString } from './utils';
 
-const porterUri: Record<string, string> = {
+const defaultPorterUri: Record<string, string> = {
   mainnet: 'https://porter.nucypher.community',
   tapir: 'https://porter-tapir.nucypher.community',
-  oryx: 'https://porter-oryx.nucypher.community',
   lynx: 'https://porter-lynx.nucypher.community',
```

Comment:
We're trying to deprecate the `.community` domain in favor of the `.io` domain.

```suggestion
  mainnet: 'https://porter.nucypher.io',
  tapir: 'https://porter-tapir.nucypher.io',
  lynx: 'https://porter-lynx.nucypher.io',
```

Reply:
addressed in #555